pkgs/applications/science/logic/klee/klee-uclibc.nix

```diff
@@ -88,13 +88,19 @@ in stdenv.mkDerivation rec {
 
   makeFlags = ["HAVE_DOT_CONFIG=y"];
 
+  enableParallelBuilding = true;
+
+  passthru.updateScript = nix-update-script {
+    extraArgs = [ "--version-regex" "v(\d\.\d)" ];
+  };
+
   meta = with lib; {
```

Comment:
Did `mainProgram` get missed?

Reply:
There is no mainProgram in this derivation but there is in the other one.